Makefile

```diff
@@ -55,22 +55,25 @@ helm-test:
 helm-lint:
 	cd config/helm && ./testing/lint.sh
 
+create-kind-cluster:
+	kind create cluster --config ./src/testing/kuttl/oneagent/kind.yaml
+
+delete-kind-cluster:
+	kind delete cluster --name kind
```

Comment:
It can be removed because it's not used now.

Reply:
Removed